tardis/io/model/stella.py

```diff
@@ -74,7 +75,7 @@ def read_stella_model(fname):
     data = pd.read_csv(
         fname,
         delim_whitespace=True,
-        skiprows=DATA_START_ROW,
+        skiprows=DATA_START_ROW + 1,
```

Comment:
Does this mean that DATA_START_ROW should start with +1 to begin with?

Reply:
what's with this?